packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -43,19 +43,19 @@ export default class UIForm extends React.Component {
 	 * @param schema The field schema
 	 * @param value The new value
 	 */
-	onChange(event, schema, value) {
+	onChange(event, { schema, value }) {
 		const {
 			formName,
 			onChange,
 			properties,
 			customValidation,
 		} = this.props;
 		const error = validateValue(schema, value, properties, customValidation);
-		onChange(formName, schema, value, error);
+		onChange(event, { formName, schema, value, error });
 
 		const { triggers } = schema;
 		if (triggers && triggers.includes(TRIGGER_AFTER)) {
-			this.onTrigger(event, TRIGGER_AFTER, schema, value, properties);
+			this.onTrigger(event, { type: TRIGGER_AFTER, schema, value, properties });
```

Comment:
why not giving formName ? Why parameters are so différents between onChange and onTrigger callback